cli/src/commands.rs

```diff
@@ -16,31 +16,31 @@
 */
 
 extern crate chrono;
-use crate::{base_cli::BaseCli, trusted_commands::TrustedArgs, Cli};
+use crate::{base_cli::BaseCommand, trusted_cli::TrustedCli, Cli};
 use clap::Subcommand;
 
 #[cfg(feature = "teeracle")]
-use crate::oracle::OracleSubCommand;
+use crate::oracle::OracleCommand;
 
 #[derive(Subcommand)]
 pub enum Commands {
 	#[clap(flatten)]
-	Base(BaseCli),
+	Base(BaseCommand),
 
 	/// trusted calls to worker enclave
 	#[clap(after_help = "stf subcommands depend on the stf crate this has been built against")]
-	Trusted(TrustedArgs),
+	Trusted(TrustedCli),
```

Comment:
This one was the hardest, I was not sure if we should just call it `Command` too?

Reply:
I think it should be `TrustedCli`, my reasoning:

If we simply call it `Command`, then it would look like this:
```rust
pub struct Command {
	/// targeted worker MRENCLAVE
	#[clap(short, long)]
	pub(crate) mrenclave: String,

	// other fields omitted 

	#[clap(subcommand)]
	pub(crate) command: TrustedCommand,
}
```
`Command.command` looks weird imho, so in that case the field name should be changed but changing it to anything else would make it confusing.

---
If we called it `TrustedCommand` then it would be even more confusing as well (basically we would reusing a type name from a different namespace for a different thing).

Reply:
Yes, I agree with what struck me however when I dug into substrate was:

```rust
struct TryRuntimeCmd {
    #[clap(short, long)]
    pub (crate) some_arg: String
    
    // ...

  #[clap(subcommand)]
   pub b(crate) command: Command,
}
```

But I still think ours is more accurate, as we really just proxy the stuff to the CLI for trusted commands.